dbus-i2c.py

```diff
@@ -71,7 +71,12 @@ def initDBusServices():
             i2cBus = deviceConfig['bus']
             i2cAddr = deviceConfig['address']
             updateInterval = deviceConfig['updateInterval']
-            device = constructor(dbusConnection(), i2cBus, i2cAddr)
+            if deviceConfig['module'] == 'ina226_service':
+                shuntRes = deviceConfig['shuntRes']
```

Comment:
Agreed. This will make sense at least for the configuration file.
There is an open question for me: Will it make sense to use these addtional parameters for INA219 also? I have no device with that chip so I have no experience on it.

Reply:
Yes, it does. I've added support here: https://github.com/pulquero/dbus-i2c/pull/3